playwright-test/global-setup.js

```diff
@@ -0,0 +1,16 @@
+// global-setup.js
+const { bsLocal, BS_LOCAL_ARGS } = require('./fixtures');
+let localStarted = false;
+const { promisify } = require('util');
+const sleep = promisify(setTimeout);
+module.exports = async (config) => {
+  console.log('Starting BrowserStackLocal ...');
+  // Starts the Local instance with the required arguments
+  bsLocal.start(BS_LOCAL_ARGS, (callback) => {
+    console.log('BrowserStackLocal Started');
+    localStarted = true;
+  });
+  while (!localStarted) {
+    await sleep(1000);
+  }
```

Comment:
Is this wait required? 
What if local fails to start?

Reply:
Throwing an error now.

Reply:
Should we exit if we fail to start the BrowserStack local? I haven't exited currently, because it also throws an error if there is any existing binary running on your system.